cypress/support/commands.ts

```diff
@@ -183,6 +183,28 @@ Cypress.Commands.add('keyboardControls', (triggerKeys: any = {}, count = 1) => {
   }
 });
 
+/**
+ * Intercept all requests and return
+ * @param {array} intercepts - Array of intercepts to return
+ * return {array} - Array of intercepted request strings
+ * return {string} - Intercepted request string
+ */
+Cypress.Commands.add('interceptAny', (method = '/GET/POST/PUT/PATCH/', url = '/v1/*', wait = false, timeout = 10000) => {
+  if (Array.isArray(url)) {
```

Comment:
Let's avoid logic in the test, especially commands. It's important to be clear in an assertion that if you want a single command or more commands, there's 2 different cases.
You man still use the single one in the array case.
Also the name is kinda confusing, because you are clearly intercepting the request, so maybe something like `interceptRequest` and `interceptAllRequests` (I added the `all` because just a `s` it's easy to miss).